pages/api/awards/collect.js

```diff
@@ -0,0 +1,50 @@
+import prisma from '@prisma/index'; // Prisma client
+import { getSession } from 'next-auth/client'; // Session handling
+
+// Get awards
+export default async (req, res) => {
+  // Collect session from request
+  const session = await getSession({ req });
+
+  //TODO: add functionality of getting awards by performance id
+
+  // If session does not exists
+  if (!session) {
+    return res.status(401).end();
+  }
+
+  // Collect all awards from database
+  const awards = await getAwards();
+
+  return res.json(awards);
+};
+
+export const getAwards = async () => {
+  // Collect awards with performanceID
+  // TODO: perhaps add flag to include the performances
```

Comment:
Do we need to retrieve awards for only certain performances

Reply:
What do you mean?

Reply:
Currently, when we retrieve an award, we always return the associated performance. Are there cases where we don't want to send this data or do we always want to send it

Reply:
Let's always send it.  I see no reason why not to include it.